samples/tab-add-in-combined/nodejs/README.md

```diff
@@ -0,0 +1,181 @@
+
+**IMPORTANT**: To stop debugging and uninstall the app, select **Run** | **Stop Debugging** in Visual Studio Code.
+
+### Run both the Teams tab and the Outlook add-in at the same time
+
+1. First, *make sure Outlook desktop is closed.*
+3. In Visual Studio Code, open the Teams Toolkit.
+4. In the **ACCOUNTS** section, verify that you're signed into Microsoft 365.
+5. Select **View** | **Run** in Visual Studio Code. In the **RUN AND DEBUG** drop down menu, select the option **Launch App and Add-in Outlook Desktop (Edge Chromium)** and then press F5. The project builds and a Node dev-server window opens to host the add-in. The tab app is hosted in the Visual Studio Code terminal. This process may take a couple of minutes. Eventually, both of the following will happen:
+
+   - Teams opens in a browser with a prompt to add your tab app. *If Teams has not opened by the time Outlook desktop opens, then automatic sideloading has failed. You can manually sideload it to see both the app and the add-in running at the same time. For sideloading instructions, see [Upload your app in Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/deploy-and-publish/apps-upload).*
+   - Outlook desktop opens.
+
+6. In the Teams prompt, select **Add** and the tab will open.
+7. In Outlook, open an Outlook message compose window by either starting a new message or selecting **Reply** on a message.
+8. Be sure there is an email address in the **To** and **From** lines.
+9. Find the **Offer Discount** button on the ribbon and select it. You will next see the following prompt. Select OK. This prompt appears whenever the add-in is run in debugging mode.
+
+     ![Dialog titled "WebView Stop On Load" with text reading "To debug the webview, attach VS Code to the webview instance using the Microsoft Debugger for Edge extension, and click OK to continue. To prevent this dialog from appearing in the future, click Cancel." Below the text are buttons labelled OK and Cancel.](Images/WebView_Stop_On_Load_dialog.JPG)
+
+   A **Discounts** task pane opens.
+
+10. Create another discount in a message.
+11. On the Teams tab, select the **Refresh data** button. The new discount you created appears at the end of the grid.
+
+## Further Reading
+
+[Extend Teams apps across Microsoft 365](https://learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/overview)
```

Comment:
Please add one more link related to img tag. Please refer our repository readme files.

Reply:
Will cover in different new PR

Reply:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/965